jsyntrax/src/main/java/org/atpfivt/jsyntrax/styles/TitlePosition.java

```diff
@@ -4,27 +4,108 @@
     /**
      * Top Left.
      */
-    tl,
+    tl {
```

Comment:
Do you feel the 'smell' of this code? Too many repetitive lines.

By using `switch` instead of series of `if`s in `SVGCanvasBuilder` (see my comment there) you can get rid of the necessity of getters.

However, if you _did_ need to use them, you could have make the code no less than three times shorter. There can be many approaches, here's the simplest: declare 

```java
private final boolean left;
private final boolean top;
....
```

initialize them in enum constructor, access them using getters that are not needed to be overriden for each enum